docs/conf.py

```diff
@@ -0,0 +1,7 @@
+from sphinx_pyproject import SphinxConfig
+
+config = SphinxConfig("../pyproject.toml", style="poetry", globalns=globals())
+project = config.name
+version = config.version
+author = config.author
+description = config.description
```

Comment:
These three will already be defined. All attrs of `config` are also added to the global scope. The only reason to set `project = config.name` is because there is no `config.project`